src/amber/websockets/client_sockets.cr

```diff
@@ -36,8 +36,8 @@ module Amber
       private def heartbeat
         spawn do
           @@heartbeat_started = true
-          @@client_sockets.each_value(&.beat)
           sleep BEAT_INTERVAL
+          @@client_sockets.each_value(&.beat)
           heartbeat
```

Comment:
the spawn is safe, but using a loop is better - done in [nick/websockets_heartbeat](https://github.com/Amber-Crystal/amber/compare/nick/websockets_pubsub...nick/websockets_heartbeat?expand=1) branch